thorlcr/msort/tsorts.cpp

```diff
@@ -792,10 +792,27 @@ class CThorSorter : public CSimpleInterface, implements IThorSorter, implements
         stopping = false;
         useTLS = queryMtls();
 #ifdef _CONTAINERIZED
-        traceLevel = getComponentConfigSP()->getPropInt("logging/@detail", 0);
+        traceLevel = getComponentConfigSP()->getPropInt("logging/@detail", InfoMsgThreshold);
```

Comment:
until the default logging level is lowered, this is still below the default.
I would introduce a constant somewhere like MyVerboseLoggingThreadhold=120 for now
can switch to ExtraneousMsgThreshold when default level changed.